### ISSUE TITLE
Upgrade to Go 1.16

### ISSUE THREAD

**Issue by grzkv** [bookingcom/nanotube#121]
Upgrade to the newly released version of Go 1.16.

**Comment by dgryski** [comment]
You can fix this once you're on 1.16

```
pkg/in/udp.go
severity:error rule:Users.dgryski.go.src.github.com.dgryski.semgrep-go.use-net-errclosed: Use errors.Is(err, net.ErrClosed) instead

28:			if strings.Contains(err.Error(), "use of closed network connection") {
```

**Comment by grzkv** [comment]
@dgryski Thanks for the input. Will definitely do that!
We'll probably do it as part of more general #88.

**Comment by grzkv** [comment]
Error type check fix moved to #124
